ibm_mq/assets/configuration/spec.yaml

```diff
@@ -141,6 +141,13 @@ files:
         value:
           example: false
           type: boolean
+      - name: queue_manager_timezone
+        description: |
+          Time zone of the queue manager in IANA/Olson timezone format, if it differs from the host system's time zone.
```

Comment:
Does it matter if its different from the host system's timezone at all? the default is UTC so even if the queue manager and the host are on the same timezone, if it's not UTC, there will be a mismatch.

Reply:
That's a good point. I'll reword it to say, "if it differs from UTC".